### PR TITLE
Support customization of the OpenAPI UI with a monitored CSS file.

### PR DESCRIPTION
**Pull request by mrglavas**
Signed-off-by: Michael Glavassevich <mrglavas@ca.ibm.com>

### CODE REVIEW

File: dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/CustomCSSProcessor.java [hunk 1]
```diff
@@ -0,0 +1,291 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
```
Comment (navidsh) [inline dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/CustomCSSProcessor.java:2]: 2018?

Comment (navidsh) [reply dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/CustomCSSProcessor.java:2]: Can you please change it across the PR?

Comment (mrglavas) [reply dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/CustomCSSProcessor.java:2]: It's not new code. This class and the others in this PR comes from the openapi-3.0 feature with some minor changes.

**Review by navidsh — APPROVED** [review]
LGTM

**Pull request merged**